kci_build

```diff
@@ -261,6 +261,107 @@ class cmd_list_kernel_configs(Command):
         return True
 
 
+class cmd_init_bmeta(Command):
+    help = "Create initial bmeta.json"
+    args = [Args.kdir]
+    opt_args = [Args.output, Args.build_config,
+                Args.tree_name, Args.tree_url, Args.branch,
+                Args.build_env, Args.arch]
+
+    def __call__(self, configs, args):
+        if args.build_config:
+            conf = configs['build_configs'][args.build_config]
+            tree_name = conf.tree.name
+            tree_url = conf.tree.url
+            branch = conf.branch
+        else:
+            tree_name = args.tree_name
+            tree_url = args.tree_url
+            branch = args.branch
+        if not all((tree_name, tree_url, branch)):
+            print("""\
+Invalid arguments, either of these 2 sets are possible:
+   --tree-name, --tree-url and --branch
+or
+   --build-config (the tree and branch are read from the YAML config)\
```

Comment:
With this code, I get:
```
$ ./kci_build init_bmeta
Invalid arguments, either of these 2 sets are possible:
   --tree-name, --tree-url and --branch
or
   --build-config (the tree and branch are read from the YAML config)
```
If I remove the backslash:
```
$ ./kci_build init_bmeta

Invalid arguments, either of these 2 sets are possible:
   --tree-name, --tree-url and --branch
or
   --build-config (the tree and branch are read from the YAML config)
```
So I put it there to avoid having this superfluous blank line on the output, but still make the multi-line string appear in the code like it would appear on the output.